ocs_ci/utility/operators.py

```
@@ -275,10 +275,50 @@ def _customize_operatorgroup(self, operatorgroup_data: dict):
+        cmd = (
+            f"oc get operatorgroup -n {self.namespace} --no-headers 2>/dev/null | wc -l"
+        )
+        try:
+            result = exec_cmd(cmd, shell=True)
+            count = int(result.stdout.decode("utf-8").strip())
+            if count > 0:
+                logger.info(
+                    f"Found {count} existing OperatorGroup(s) in namespace {self.namespace}"
+                )
+            return count > 0
+        except Exception as e:
```

Ruff (0.15.13)

[warning] Do not catch blind exception: `Exception`

(BLE001)